<div>To help account for the uncertainty in ancestry inference we sampled chromosome paintings 10 times for each individual. We use a binomial likelihood to test whether the ancestry at each SNP along the genome differs from expectations. Specifically, for each recipient haplotype, we infer the expected proportion of ancestry from each non-self ancestry region from their genome-wide ancestry paintings. At each position, we fit a deviation parameter, <span class="math ltx_Math v1">\(\beta\)</span>, that best explains the sampled ancestries at that SNP across all individuals. <span class="math ltx_Math v1">\(\beta\)</span> can be thought of as the increase (or decrease) in expected ancestry proportions across individuals on the logit scale that best explains the proportions observed at a SNP compared to the genome-wide average. To generate a <span class="math ltx_Math v1">\(P\)</span> value, we compared the likelihood ratio test statistic to a <span class="math ltx_Math v1">\(\chi^{2}\)</span> distribution. At each SNP in the genome, this test gives a <span class="math ltx_Math v1">\(P\)</span> value and estimate of <span class="math ltx_Math v1">\(\beta\)</span> for each of the 7 non-self ancestries tested.</div><div>


Modelling ancestry in this way has the advantage that it adjusts for an individualâ€™s specific ancestry proportions. The likelihood is a product across individuals which assumes that individuals are independent given the underlying ancestry proportions. Quantile-quantile (Q-Q plots) of the observed test statistics typically show high levels of inflation (Supplementary Fig.&nbsp;<span class="au-ref raw v1">\ref{fig:suppfulaQQ}</span>). There are at least three explanations for this inflation. Firstly, due to shared ancestral histories among haplotypes from the same population, the paintings are unlikely to be independent between samples in a population, so the assumption in the likelihood that the samples are independent is likely to be false. Moreover the demographic process that has given rise to the ancestry paintings is more complex than the model, leading to local variations in paintings that are not captured by the null. Secondly, the chromosome painting model explicitly uses correlations in ancestry locally in the genome (i.e. linkage disequilibrium) meaning that tests are not independent along the genome, as shown by the autocorrelation plot in Supplementary Figure&nbsp;<span class="au-ref raw v1">\ref{fig:suppfulaQQ}</span>. Finally, adaptive introgression may have a weak effect genome-wide. We discuss below alternative approaches that model the variance (as well as the expectation) in ancestry proportions across the genome. However, these limitations mean that it is difficult to estimate a genome-wide threshold for significance in the traditional sense (i.e. for defining false positive/negative rates), so we view&nbsp;<span class="math ltx_Math v1">\(P\)</span> values as a ranking rather than absolute terms.</div><h2 data-label="540703" class="ltx_title_subsection">


Ancestry deviations in the Fula identify known targets of selection</h2><div>


We illustrate our approach with an analysis of genomes from the Fula ethnic group from The Gambia. The Fula (also known as Fulani) are historically nomadic pastoralists spread across West Africa who we inferred in our previous analysis&nbsp;<cite class="ltx_cite raw v1">\citep{Busby2016AdmixtureAfrica}</cite> to have experienced an admixture event involving a largely Eurasian source (0.19 admixture proportion) mixing with a West African source around 1,800 years ago (239CE (95% CI = 199BCE-486CE)). (Admixture events across all analysis populations inferred previously&nbsp;<cite class="ltx_cite raw v1">\citep{Busby2016AdmixtureAfrica}</cite> are summarised in Supplementary Figure&nbsp;<span class="au-ref raw v1">\ref{fig:admOverview}</span>). This event introduced a significant amount of Eurasian ancestry into the Fula which is relatively easy to identify given that Eurasian populations split and drifted from African populations over 50kya&nbsp;<cite class="ltx_cite raw v1">\citep{Mallick2016ThePopulations}</cite>. They thus have haplotypes that are easier to differentiate from each other, making our approach relatively well powered to detect deviations in ancestry proportions.</div><div>


In the Fula, the region of the genome with the lowest proportion of Eurasian ancestry (<span class="math ltx_Math v1">\(-log_{10}P=10\)</span>, <span class="math ltx_Math v1">\(\beta\)</span>=-4 across all Fula individuals) is on chromosome 1 and contains the Duffy Antigen Receptor <i>DARC</i> gene [Fig. <span class="au-ref raw v1">\ref{fig:fig2}</span>a] whilst the region with the highest level of Eurasian ancestry (<span class="math ltx_Math v1">\(-log_{10}P=15\)</span>, <span class="math ltx_Math v1">\(\beta\)</span>=2) is on chromosome 2 and contains the <i>LCT</i> and <i>MCM6</i> genes [Fig. <span class="au-ref raw v1">\ref{fig:fig2}</span>b]. Polymorphisms in DARC form the basis of the Duffy blood group system, and the Duffy null mutation, which provides resistance to <i>Plasmodium vivax</i> malaria&nbsp;<cite class="ltx_cite raw v1">\citep{Miller1976TheBlacks}</cite> is almost fixed across Africa and absent outside&nbsp;<cite class="ltx_cite raw v1">\citep{Howes2011TheGroup}</cite>. Our observation of less Eurasian ancestry than expected at this locus suggests that African haplotypes have been beneficial at this locus after the admixture event that brought the Eurasian ancestry into the Fula; that is, within the last 2,000 years. Similar effects at this locus have been observed before: based on an analysis of allele frequencies, selection following admixture between Austronesians and sub-Saharan Bantus is likely to have driven frequencies of the Duffy null mutation to higher than expectations based on admixture proportions in the Malagasy of Madagascar&nbsp;<cite class="ltx_cite raw v1">\citep{Hodgson2014NaturalMadagascar}</cite>, and a recent study of Sahel populations showed an excess of African ancestry at this locus in Sudanese Arabs and Nubians&nbsp;<cite class="ltx_cite raw v1">\citep{Triska2015ExtensiveBelt}</cite>.</div><div></div>