<div>In the Ju/hoansi, the top region of differentiation in the genome involves an increase in East Africa Niger-Congo ancestry at 26.1-26.6Mb on chromosome 10 (<span class="math ltx_Math v1">\(-log_{10}P=17.14\)</span>, <span class="math ltx_Math v1">\(\beta=2.07\)</span>) which contains <i>MYO3A</i>, a gene involved in hearing&nbsp;<cite class="ltx_cite raw v1">\citep{Walsh2002FromDFNB30}</cite>, and <i>GAD2</i>, which has been associated with decreased body mass index in Europeans&nbsp;<cite class="ltx_cite raw v1">\citep{Boesgaard2007ASubjects}</cite>. We also observe a significant increase (<span class="math ltx_Math v1">\(-log_{10}P=11.48\)</span>, <span class="math ltx_Math v1">\(\beta=1.86\)</span>) of East Africa Afroasiatic ancestry at 99.08-99.20Mb on chromosome 13, a region containing <i>FARP1</i> and <i>STK24</i>. <i>FARP1</i> is thought to be a member of the band 4.1 protein superfamily&nbsp;<cite class="ltx_cite raw v1">\citep{Koyano1997MolecularFactors}</cite>, a major structural component of red blood cell membranes&nbsp;<cite class="ltx_cite raw v1">\citep{Conboy1986MolecularSkeleton}</cite>. We also see an increase in West African Niger-Congo ancestry at 47.65-47.76Mb on chromosome 6 a region containing <i>CD2AP</i>, <i>OPN5</i>, <i>GPR111</i>, and <i>GPR115</i> genes, and a large increase in Eurasian ancestry at <i>DISP1</i> on chromosome 1 (<span class="math ltx_Math v1">\(-log_{10}P=10.70\)</span>, <span class="math ltx_Math v1">\(\beta=2.88\)</span>).</div><div>


Eight populations [Fig.&nbsp;<span class="au-ref raw v1">\ref{fig:figSharedAncestry}</span>] showed evidence of significant ancestry deviation at a short haplotype in the middle of the <i>LDB2</i> gene on chromosome 4. All 8 of these populations come from East Africa and are either Nilo-Saharan or Afroasiatic speakers [Supplementary Table 1; Supplementary Fig.&nbsp;<span class="au-ref raw v1">\ref{fig:figLDB2}</span>]. We sought to characterise the nature of this signal further by combining data across all 235 individuals from these 8 populations (Sudanese, Somali, Anuak, Gumuz, Ari, Tigray, Amhara and Oromo) and the two other populations from the region with nominal evidence of deviation (Afar and Wolayta). In all cases, we observed a signal of increased Khoesan ancestry [Supplementary Fig.&nbsp;<span class="au-ref raw v1">\ref{fig:figLDB2}</span>a] and localised this to increased copying specifically from the Ju/hoansi, Xun, and Amaxhosa of southern Africa [Supplementary Fig.&nbsp;<span class="au-ref raw v1">\ref{fig:figLDB2}</span>d]. This differs considerably from a randomly selected SNP 1.5Mb downstream, with a more usual copying profile and where the Luhya and several Eurasian groups were typically copied more. <i>LDB2</i> codes for a LIM-domain binding protein, <i>LDB2</i>, which is a transcriptional regulator and has been linked to carotid atheroschlerosis&nbsp;<cite class="ltx_cite raw v1">\citep{Mm2014LimAtherosclerosis.}</cite>.</div><h2 data-label="677361" class="ltx_title_subsection">Considerations of our approach to infer adaptive gene-flow</h2><div>


Whilst admixture appears to be an almost universal characteristic of the populations that we use in this analysis, we only find evidence of substantial adaptive gene-flow in three populations: the Fula and Jola from the Gambia and the Ju/hoansi from Namibia. Importantly, we infer significant recent admixture involving diverged source populations in these three groups, from Eurasian-like sources in the Gambians, and East African Asiatic-like source in the Ju/hoansi [Supplementary Fig. <span class="au-ref raw v1">\ref{fig:admOverview}</span>]&nbsp;<cite class="ltx_cite raw v1">\citep{Busby2016AdmixtureAfrica}</cite>. Our method is likely to have greater power to detect deviations in these scenarios. As such, it is perhaps not surprising that across populations from the Central West African ancestry region, where we previously inferred admixture events involving mostly closely related African sources&nbsp;<cite class="ltx_cite raw v1">\citep{Busby2016AdmixtureAfrica}</cite> we infer a single example of adaptive gene-flow in the Namkam from Ghana at <i>FZD1</i> on chromosome 8 [Fig. <span class="au-ref raw v1">\ref{fig:figSharedAncestry}</span>]. It is important to note that even though all Gambian populations in our analysis had similar sample sizes and were painted using exactly the same procedure as the Jola and Fula, we do not see similar numbers of local ancestry changes in them, suggesting that such deviations are not linked to our chromosome painting procedure.</div><div>


In addition to working with populations that have substantial recent admixture, there are further considerations to the results that come from our approach. The local ancestry inference procedure that we have used&nbsp;<cite class="ltx_cite raw v1">\citep{Lawson2012InferenceData}</cite> is based on a fully probabilistic framework. To access uncertainty in local ancestry with the paintings, we generated 10 realisations (sampled paintings) of the painting algorithm and inferred ancestry proportions from these paintings. Because of the stochastic nature of such an approach, we are unlikely to capture the full uncertainty in local ancestry, but chose this approach based on computational requirements and the use of a similar approach in previous admixture inference work&nbsp;<cite class="ltx_cite raw v1">\citep{Hellenthal2014AHistory}</cite>. Nevertheless, future work to optimise the computation issues of storing the <span class="math ltx_Math v1">\(n*n\)</span> posterior copying probability matrix at each locus in the genome will likely provide a clearer assessment of local ancestry uncertainty. We used a binomial likelihood model, which takes each ancestry separately and models genome-wide proportions against local deviations. Using this model, we found no evidence that more than one ancestry increases or decreases in concert at a locus, at least in the Fula [Fig. <span class="au-ref raw v1">\ref{fig:figVarCovar}</span>]. Modelling changes across all ancestries at the same time, for example by using a multi-variate normal model, might provide further insights into part of the genome where we observe co-deviations.</div>