<div></div><h1 class="ltx_title_section">Introduction</h1><div>


The source of adaptive genetic variation remains an important question in evolutionary biology<cite class="ltx_cite raw v1">\cite{Hedrick2013AdaptiveVariation}</cite>. In the canonical view of evolution, natural selection causes a change in the frequency of a mutation that was either present before the introduction of selection, so-called standing variation, or a new mutation that arose after the selective pressure began<cite class="ltx_cite raw v1">\cite{Hermisson2005SoftVariation}</cite>. A third potential source of novel genetic variation on which natural selection can act is via gene-flow resulting from an inter or intra species admixture event. Examples of such adaptive introgression include the exchange of mimicry adaptations amongst Heliconius butterfly species<cite class="ltx_cite raw v1">\cite{Consortium2012ButterflySpecies}</cite>, the transfer of insecticide resistance genes between sibling Anopheles mosquito species<cite class="ltx_cite raw v1">\cite{Clarkson2014AdaptiveIsolation}</cite>, and the spread of pesticide resistance across populations of house mice<cite class="ltx_cite raw v1">\cite{Staubach2012GenomeMusculus}</cite>.</div><div>

In the human lineage, recent genomic studies have shown that our demographic history is complex, involving population merges as well as splits<cite class="ltx_cite raw v1">\cite{Green2010AGenome,Patterson2012AncientHistory,Prufer2014TheMountains,Hellenthal2014AHistory,Fu2015AnAncestor}</cite>. These merges, also known as admixture or introgression events, open the door to the transfer of adventageous alleles. The trysts between archaic <i>Homo sapiens</i> and Neanderthals over fifty thousand years ago (kya) transfered beneficial alleles from Neanderthals into the human lineage <cite class="ltx_cite raw v1">\cite{Green2010AGenome,Patterson2012AncientHistory,Prufer2014TheMountains,Fu2015AnAncestor,Abi-Rached2011TheHumans,Gittelman2016ArchaicEnvironments}</cite>. While genomic regions with elevated archaic ancestry have been observed, it has also been possible to identify regions that are devoid of archaic ancestry and where it is thought that archaic alleles at these loci have been at a disadvantage<cite class="ltx_cite raw v1">\cite{Sankararaman2014TheHumans,Racimo2015EvidenceHumans}</cite>.</div><div>


Despite the prevalence of more recent admixture events over the past 50 kya of human history, relatively few instances of adaptation from recent merges have been found. Notable known examples include the exchange of high altitude adaptations between the ancestors of Sherpa and Tibetans<cite class="ltx_cite raw v1">\cite{Jeong2014AdmixtureTibet}</cite> and the spread of the Plasmodium vivax malaria-proecting Duffy null mutation in Madagascar as result of gene-flow from mainland African Bantu-speakers<cite class="ltx_cite raw v1">\cite{Hodgson2014NaturalMadagascar}</cite>.</div><div>


Within Africa, multiple studies using different techniques have shown admixture to be a common theme in the recent history of the continent<cite class="ltx_cite raw v1">\cite{Pagani2012EthiopianPool,Schlebusch2012GenomicHistory,Pickrell2012TheAfrica,Pickrell2014AncientAfrica,Pickrell2014TowardDNA,Busby2016AdmixtureAfrica,Patin2017DispersalsAmerica}</cite>. As a result, the genomes of individuals from these populations contain segments which derive from multiple ancestries. Inferring whether gene-flow has assisted adaptation requires inferring how these ancestries change locally along the genome. Although a number of strategies exist for local ancestry inference<cite class="ltx_cite raw v1">\cite{Price2009SensitivePopulations,Baran2012FastPopulations,Brisbin2012PCAdmix:Populations,Maples2013RFMix:Inference}</cite>, most are designed to distinguish at best continental-level ancestry from a small number of reference populations. To identify whether haplotypes of specific ancestry have potentially moved within, as well as from outside of, Africa, the approach we develop here involves sampling from a Hidden Markov Model to identify the likely donor haplotype from a large set of reference genomes. We use this chromosome painting approach to assign a donor ancestry label (which can be at individual, population, region, or continental level) at each locus in the genome to all recipient individuals. Iterating this process incorporates ancestry assignment uncertainty and we infer the ancestry proportions from different donor groups across the genome.</div><div>Here we use ChromoPainter&nbsp;<cite class="ltx_cite raw v1">\cite{Lawson2012InferenceData}</cite> to infer ancestry across the genomes of 3,283 individuals from a set of 60 donor groups, and use this analysis to ask a simple question: across individuals in these admixed populations, are there regions of the genome where ancestry is significantly deviated away from genome-wide expectations? We construct a statistical model to test for significant deviations. We interpret deviations in local ancestry as possibly resulting from natural selection acting to either increase the frequency of the introgressed haplotype (positive selection) or prevent it from replacing an established haplotype (negative selection). We highlight specific examples where ancestry deviations align with known targets of selection, describe patterns of selection across the data, and discuss the challenges in using approaches of this kind for detecting adaptive admixture.</div><h1 data-label="780116" class="ltx_title_section">
Results and Discussion</h1><h2 data-label="513143" class="ltx_title_subsection">


Inferring local ancestry</h2><div>


We used a published<cite class="ltx_cite raw v1">\cite{Busby2016AdmixtureAfrica}</cite> dataset containing computationally phased haplotypes from 3,283 individuals, from 60 worldwide populations, typed at 328,176 high quality genome-wide SNPs [Supplementary Fig. <span class="au-ref raw v1">\ref{fig:supppopsmap}</span>]. We grouped individuals from these populations into 8 separate ancestry regions, based on genetic and ethno-linguistic similarity, as described previously<cite class="ltx_cite raw v1">\cite{Busby2016AdmixtureAfrica}</cite> [Fig. <span class="au-ref raw v1">\ref{fig:fig1}</span>a]. We painted chromosomes from these populations using ChromoPainter with donors which did not include any closely related populations, here defined as individuals from within their own ancestry region. Therefore, for each haplotype, there are seven non-local ancestries (or colours of paint) from which they can copy.</div><h2 data-label="661494" class="ltx_title_subsection">



Identifying changes in local ancestry</h2><div>


Our approach for inferring ancestry deviations is based on the idea that ancestry tracts shared between two populations will reveal a signal of admixture<cite class="ltx_cite raw v1">\cite{Hellenthal2014AHistory}</cite>, but will be randomly distributed amongst the genomes of individuals within those populations<cite class="ltx_cite raw v1">\cite{Baird2006FishersAdmixture}</cite>[Fig. <span class="au-ref raw v1">\ref{fig:fig1}</span>b]. So, whilst individuals will have mosaic (i.e. block-like) ancestry [Fig. <span class="au-ref raw v1">\ref{fig:fig1}</span>c], when summed across all individuals in a population, ancestry proportions at specific loci (SNPs) will resemble genome-wide proportions in expectation [Fig. <span class="au-ref raw v1">\ref{fig:fig1}</span>d]. In this framework, we would expect ancestry proportions at a SNP to match genome-wide proportions, with some variation which we can model. A significant deviation away from expected ancestry proportions suggests that natural selection may have contributed to the change. We note that this will only ever allow for indirect inference of selection. This is nevertheless similar in spirit to other commonly used selection identification methods, such as the integrated haplotype score (iHS)<cite class="ltx_cite raw v1">\cite{Sabeti2002DetectingStructure,Voight2006AGenome}</cite> which scans the genomes of a population for longer than expected haplotypes based on their frequency, which are then (indirectly) inferred to have swept as a result of selection.</div><div>
\afterpage</div>