<h1 class="ltx_title_section">Results and Discussion</h1>


<h2 class="ltx_title_subsection">Inferring local ancestry</h2>


<div>We used a published<cite class='ltx_cite raw v1'>\cite{Busby2016AdmixtureAfrica}</cite> dataset containing computationally phased haplotypes from 3,283 individuals, from 60 worldwide populations, typed at 328,176 high quality genome-wide SNPs [Supplementary Fig. <span class='au-ref raw v1'>\ref{fig:supppopsmap}</span>]. We grouped individuals from these populations into 8 separate ancestry regions, based on genetic and ethno-linguistic similarity, as described previously<cite class='ltx_cite raw v1'>\cite{Busby2016AdmixtureAfrica}</cite> [Fig. <span class='au-ref raw v1'>\ref{fig:fig1}</span>a]. We painted chromosomes from these populations using ChromoPainter with donors which did not include any closely related populations, here defined as individuals from within their own ancestry region. Therefore, for each haplotype, there are seven non-local ancestries (or colours of paint) from which they can copy.</div>



<h2 class="ltx_title_subsection">Identifying changes in local ancestry</h2>


<div>Our approach for inferring ancestry deviations is based on the idea that ancestry tracts shared between two populations will reveal a signal of admixture<cite class='ltx_cite raw v1'>\cite{Hellenthal2014AHistory}</cite>, but will be randomly distributed amongst the genomes of individuals within those populations<cite class='ltx_cite raw v1'>\cite{Baird2006FishersAdmixture}</cite>[Fig. <span class='au-ref raw v1'>\ref{fig:fig1}</span>b]. So, whilst individuals will have mosaic (i.e. block-like) ancestry [Fig. <span class='au-ref raw v1'>\ref{fig:fig1}</span>c], when summed across all individuals in a population, ancestry proportions at specific loci (SNPs) will resemble genome-wide proportions in expectation [Fig. <span class='au-ref raw v1'>\ref{fig:fig1}</span>d]. In this framework, we would expect ancestry proportions at a SNP to match genome-wide proportions, with some variation which we can model. A significant deviation away from expected ancestry proportions suggests that natural selection may have contributed to the change. We note that this will only ever allow for indirect inference of selection. This is nevertheless similar in spirit to other commonly used selection identification methods, such as the integrated haplotype score (iHS)<cite class='ltx_cite raw v1'>\cite{Sabeti2002DetectingStructure,Voight2006AGenome}</cite> which scans the genomes of a population for longer than expected haplotypes based on their frequency, which are then (indirectly) inferred to have swept as a result of selection.</div>
\afterpage