<h2 data-label="222893" class="ltx_title_subsection">The genome-wide distribution of ancestry deviations in the Fula</h2><div>


We note that because Eurasian and Khoesan haplotypes are likely to be more diverged from those of other sub-Saharan African groups we are better able to characterise them and so have more power to identify deviations concerning these ancestries. Indeed, over half (21) of the top 40 ancestry deviations observed in the Fula involve Eurasian ancestry [Table S1; Fig. <span class="au-ref raw v1">\ref{fig:suppfulafig}</span>], including increases of Eurasian ancestry across the <i>HLA</i> region on chromosome 6 (<span class="math ltx_Math v1">\(-log_{10}P=12.38\)</span> ,<span class="math ltx_Math v1">\(\beta=1.29\)</span>), a group of <i>DSC</i> genes on chromosome 18 (<span class="math ltx_Math v1">\(-log_{10}P=13.55\)</span>, <span class="math ltx_Math v1">\(\beta=1.35\)</span>), and at <i>APOL</i> genes on chromosome 22 (<span class="math ltx_Math v1">\(-log_{10}P=13.01\)</span>, <span class="math ltx_Math v1">\(\beta=1.32\)</span>). Of the non-Eurasian signals, of note are a significant increase in Central West African ancestry across the <i>GYP</i> genes on chromosome 4 [Fig. <span class="au-ref raw v1">\ref{fig:fig2}</span>c] (<span class="math ltx_Math v1">\(-log_{10}P=8.96\)</span>, <span class="math ltx_Math v1">\(\beta=1.13\)</span>), encompassing a region which has recently been identified as being associated with malaria&nbsp;<cite class="ltx_cite raw v1">\citep{MalariaGenomicEpidemiologyNetwork2015ASelection,Leffler083634}</cite>, and which, based on a recent iHS analysis&nbsp;<cite class="ltx_cite raw v1">\citep{Johnson2017PatternsPopulations}</cite>, shows signs of being under natural selection. We observe an increase in Khoesan ancestry across the <i>SERPIN</i> genes on chromosome 18 [Fig. <span class="au-ref raw v1">\ref{fig:fig2}</span>d] (<span class="math ltx_Math v1">\(-log_{10}P=19.32\)</span>, <span class="math ltx_Math v1">\(\beta=2.98\)</span>), one of which, <i>SERPINB11</i>, has previously been implicated as a target of natural selection in pastoralist groups in Africa&nbsp;<cite class="ltx_cite raw v1">\citep{Seixas2012LossInteractions}</cite>.</div><div>


We next looked at the evidence for covariance in ancestry deviation at SNPs across the genome. As expected, in all cases we found negative covariance in ancestry at SNPs where we observed a significant deviation signal [Fig. <span class="au-ref raw v1">\ref{fig:figVarCovar}</span>a]. When one ancestry increases in frequency at a locus, it must be at the detriment to other ancestries. This association was more pronounced for more closely related ancestries. For example, the largest negative relationships were seen between East African groups [red boxes in Fig. <span class="au-ref raw v1">\ref{fig:figVarCovar}</span>a]. Interestingly, when we looked at the distribution of <span class="math ltx_Math v1">\(\beta\)</span>s at significant SNPs, there was a general trend towards signals of ancestry increases , except in the case of Eurasian (yellow) and Central West African (light blue) ancestry [Fig. <span class="au-ref raw v1">\ref{fig:figVarCovar}</span>c].</div><div></div>